contracts/Bulker.sol

```diff
@@ -137,4 +145,11 @@ contract Bulker {
         (bool success, ) = to.call{ value: amount }("");
         if (!success) revert FailedToSendEther();
     }
+
+    /**
+     * @notice Claim reward for a user
+     */
+    function claimReward(address src, bool shouldAccrue) internal {
+        ICometRewards(rewards).claim(comet, src, shouldAccrue);
```

Comment:
`claimTo` is more generic than `claim` but is more gassy due to the permission check. I think for our use-cases, `claim` is sufficient, but we could always add a new action for `claimTo` if we need it.